client/jetpack-app/plans/style.scss

```diff
@@ -0,0 +1,33 @@
+@import "calypso/my-sites/plans-grid/media-queries";
+
+body {
+	background-color: #fdfdfd;
+}
+
+.jetpack-app-plans {
+	@include plan-features-layout-switcher;
+
+	.plans__header {
+		display: block;
+		overflow: auto;
+
+		.formatted-header__title,
+		.formatted-header__subtitle {
+			max-width: unset;
+		}
+
+		h1 {
+			font-family: $brand-serif;
+			font-size: $font-headline-small;
+			margin-bottom: 16px;
+		}
+
+		p {
+			font-size: $font-body;
+			color: var(--color-text-subtle);
+			text-align: center;
+			padding: 0 16px;
```

Comment:
That's a good point. The only comprehensive global spacing declarations I found were `grid-unit` https://github.com/Automattic/wp-calypso/pull/82135/commits/e1b4a6b2466c343164ccb03e3dacffb0325627a8, I'm using them instead of hardcoded values.